Australian Digital Observatory/content/the-ado-ecosystem/contents.lr

```diff
@@ -19,14 +19,52 @@ This modular ecosystem approach gives flexibility, allowing researchers to pick
 
 In designing and curating the ADO ecosystem, we take inspiration from the open source software community and [the Unix philosophy](https://medium.com/ingeniouslysimple/philosophy-of-unix-development-aa0104322491) of designing interoperable tools that each do one thing well, and from foundational tooling ecosystems such as [the R Tidyverse](https://rviews.rstudio.com/2017/06/08/what-is-the-tidyverse/#:~:text=The%20tidyverse%20is%20a%20coherent,being%20expanded%20by%20several%20contributors.), which has well demonstrated the suitability of such a structure to the academic research environment. We hope to learn from these communities, and also to contribute back to them.
 
+### Example workflows
+
+Here are some examples of research workflows that benefit from our ecosystem approach.
+
+<<<<<<< HEAD
```

Comment:
Looks great, Boyd - there's just a git diff artefact here and line 30 below. Once that's cleaned up, just go ahead and merge it :)